src/main/java/frc/robot/events/BallStored.java

```diff
@@ -18,6 +18,7 @@ public boolean check() {
 
   public Command getCommand() {
     mLogger.info("Ball Stored");
+    Storage.getInstance().increaseBallCount();
     return new StopBallStorage();
```

Comment:
general question: do the rest of our state machine commands effectively return the next state? Otherwise, returning a StopBallStorage from a getCommand of BallStored seems like violating the plain english contract of the code.

Reply:
I'm not entirely sure, and I'm not entirely sure this is the PR to fix it in either